app/views/layouts/application.html.haml

```diff
@@ -18,8 +18,7 @@
         = ' - '
       = title
 
-    = stylesheet_pack_tag 'common', media: 'all'
-    = stylesheet_pack_tag stylesheet_for_layout, media: 'all'
```

Comment:
Because `extract-text-webpack-plugin` bundles all stylesheets required by the entry point, just as did Webpack before #3332.